.github/workflows/build.yml

```
@@ -1,8 +1,8 @@
 name: Test
 on: pull_request
 jobs:
-  spellcheck:
-    name: Spellcheck
+  build:
+    name: Build
```

Can I trigger a rebuild with a noop suggestion?
```suggestion
    name: Build
```

---

Nope!